### DISCUSSION TITLE
Cannot login and Dashboard disappears

### DISCUSSION THREAD

**Discussion by alexmb2020** [koush/scrypted#281] [General]
Hi, Scrypted installed on Synology's Docker for amcrest and dahua cameras. After updating to  0.1.X cannot login, wrong password message. Reinstalled to latest 0.1.6 and did new setup, all looks good except cannot see dashboard - it was front page with all cameras. 
A few cameras not streaming video, need to select "no audio" under camera's "General".
A few questions:
Can I enable dashboard?
Can I save all settings or backup to restore it later?
Thank You!

**Comment by koush** [comment]
dashboard was removed. did not want to maintain that. use homekit, etc, for the dashboard. or someone else can take up a dashboard plugin.

**Reply by alexmb2020** [reply]
ok, tnx for quick reply.  And thank You for the great platform!
